src/alchemlyb/parsing/amber.py

```diff
@@ -199,13 +239,52 @@ def file_validation(outfile):
             invalid = True
     if invalid:
         return False
-    file_datum = FEData()
     file_datum.clambda = clambda
     file_datum.t0 = t0
     file_datum.dt = dt
     file_datum.T = T
+    file_datum.have_mbar = have_mbar
     return file_datum
 
+
+def extract_u_nk(outfile):
+    file_datum = file_validation(outfile)
+    if not file_validation(outfile):
+        return None
+    if not file_datum.have_mbar:
+        logging.error('ERROR: No MBAR energies found! Cannot parse file.')
```

Comment:
SHouldn't this raise an exception? At this point it shouldn't continue, should it?

Reply:
It should behave in the same way as the TI parser.